### DISCUSSION TITLE
Attachment within Table Row

### DISCUSSION THREAD

**Discussion by ahmadnugroho-asp** [docmost/docmost#255] [Help / Questions] [answered]
I create a new page with a table.
Try to upload file attachment in the row but the table get split as the screen below.

Its good to have the uploaded file name and link properly aligned in the table

![image](https://github.com/user-attachments/assets/95bb1fe8-451e-4906-8603-a10a8b61dbe3)

**Comment by Philipinho** [comment, marked as answer]
Fixed.
~File attachment is a block node which cannot be inserted in tables.~

**Reply by bolavefasfas** [reply]
can image be attached in the table cell ?

**Reply by Philipinho** [reply]
@bolavefasfas, no. Images are block nodes too.

**Reply by bolavefasfas** [reply]
will it be there in future, its a feature I use not in upnote, notion and excel.

**Reply by zisco67** [reply]
@Philipinho So you want to tell us, that it is (and will not be) possible to insert images in cells? This is an absolutely NO-GO! Please fix that!

**Reply by n8himmel** [reply]
Text is a block node, but text can be used in table cells. From UI standpoint these are completely different elements.

Image is a block node, but images can be used in table cells. From UI standpoint these are completely different elements.

**Reply by zisco67** [reply]
@n8himmel Sorry, but from user standpoint I really don't get your point - are you telling us, that it should be possible to use images in table cells or what?

**Reply by Philipinho** [reply]
@zisco67 This is already supported in recent versions. You can already upload attachments and images to table cells.

You don't start with supporting every feature from the on set.
